Sources/Networking/Backend.swift

```diff
@@ -155,34 +141,49 @@ class Backend {
         let loginCallback = LogInCallback(cacheKey: loginOperation.cacheKey, completion: completion)
         let cacheStatus = self.logInCallbacksCache.add(callback: loginCallback)
 
-        self.operationQueue.addCacheableOperation(loginOperation, cacheStatus: cacheStatus)
+        self.config.operationQueue.addCacheableOperation(loginOperation, cacheStatus: cacheStatus)
     }
 
     func getOfferings(appUserID: String, completion: @escaping OfferingsResponseHandler) {
-        let config = NetworkOperation.UserSpecificConfiguration(httpClient: self.httpClient,
-                                                                authHeaders: self.authHeaders,
+        let config = NetworkOperation.UserSpecificConfiguration(httpClient: self.config.httpClient,
+                                                                authHeaders: self.config.authHeaders,
                                                                 appUserID: appUserID)
         let getOfferingsOperation = GetOfferingsOperation(configuration: config,
                                                           offeringsCallbackCache: self.offeringsCallbacksCache)
 
         let offeringsCallback = OfferingsCallback(cacheKey: getOfferingsOperation.cacheKey, completion: completion)
         let cacheStatus = self.offeringsCallbacksCache.add(callback: offeringsCallback)
 
-        self.operationQueue.addCacheableOperation(getOfferingsOperation, cacheStatus: cacheStatus)
+        self.config.operationQueue.addCacheableOperation(getOfferingsOperation, cacheStatus: cacheStatus)
     }
 
     func getIntroEligibility(appUserID: String,
                              receiptData: Data,
                              productIdentifiers: [String],
                              completion: @escaping IntroEligibilityResponseHandler) {
-        let config = NetworkOperation.UserSpecificConfiguration(httpClient: self.httpClient,
-                                                                authHeaders: self.authHeaders,
+        let config = NetworkOperation.UserSpecificConfiguration(httpClient: self.config.httpClient,
+                                                                authHeaders: self.config.authHeaders,
                                                                 appUserID: appUserID)
         let getIntroEligibilityOperation = GetIntroEligibilityOperation(configuration: config,
                                                                         receiptData: receiptData,
                                                                         productIdentifiers: productIdentifiers,
                                                                         responseHandler: completion)
-        self.operationQueue.addOperation(getIntroEligibilityOperation)
+        self.config.operationQueue.addOperation(getIntroEligibilityOperation)
+    }
+
+}
+
+extension Backend {
+
+    enum QueueProvider {
+
+        static var queue: OperationQueue {
```

Comment:
I would turn this into a `func` to make it more clear that it's creating a new `OperationQueue` every time it's called.
Maybe also rename it?

```suggestion
        static func createQueue() -> OperationQueue {
```